Index: v4.27-9668/src/Mayaqua/Network.c
===================================================================
--- v4.27-9668.orig/src/Mayaqua/Network.c
+++ v4.27-9668/src/Mayaqua/Network.c
@@ -9429,7 +9429,7 @@
 		UnixSetSocketNonBlockingMode(sock->socket, true);
 	}
 
+#ifndef SSL_OP_NO_SSLv3
-#if OPENSSL_VERSION_NUMBER < 0x10100000L
 	if (sock->ssl != NULL && sock->ssl->s3 != NULL)
 	{
 		sock->Ssl_Init_Async_SendAlert[0] = sock->ssl->s3->send_alert[0];
@@ -13025,7 +13025,7 @@
 		{
 			if (client_tls == false)
 			{
+#ifndef SSL_OP_NO_SSLv3
-#if OPENSSL_VERSION_NUMBER < 0x10100000L
 				SSL_CTX_set_ssl_version(ssl_ctx, SSLv3_method());
 #else
 				SSL_CTX_set_ssl_version(ssl_ctx, SSLv23_method());
